README.md

```diff
@@ -158,16 +160,13 @@ Tasks Supported:
 - Question Answering
 - Masked Language Modeling (MLM)
```

Comment:
we could add the `task="blank"` strings here or link to the "Try It Now" docs for the listed tasks (talking about these ones https://neuralmagic.com/use-cases/#nlp)

Reply:
good one